pgctl/cli.py

```diff
@@ -208,6 +213,17 @@ def __locked_change_state(self, state):
 
         return failed
 
+    def run_pre_start_hook(self):
+        """Run the playground-wide pre-start hook, if it exists."""
+        try:
+            path = self.pgdir.join('pre-start')
+            if path.exists():
+                subprocess.check_call((path.strpath))
```

Comment:
this may look like a tuple, but you've just parenthesized a variable (wait wtf how does this work?: there's a special case in the subprocess code for a single string as the first argument)